integration_tests/pages/searchForABookingReference.ts

```diff
@@ -0,0 +1,29 @@
+import Page, { PageElement } from './page'
+
+export default class SearchForABookingReferencePage extends Page {
+  constructor() {
+    super('Search for a booking')
+  }
+
+  enterReferenceBlockOne = (number: string): void => {
```

Comment:
Could perhaps consider something like:
```
enterVisitReference = (reference: string): void => {
  const blocks = reference.split('-')

  cy.get('#searchBlock1').clear().type(blocks[0])
  cy.get('#searchBlock2').clear().type(blocks[1])
  // ...
}
```

then, in your test script, you'd just have a single call: `searchForABookingReferencePage.enterVisitReference('ab-cd-ef-gh')`